docker/Dockerfile

```diff
@@ -161,45 +162,9 @@ ENV NETWORK ${NETWORK:-Mainnet}
 ARG RPC_PORT
 ENV RPC_PORT ${RPC_PORT}
 
-# Set this to log to a file, if not set, logs to standard output
-ARG LOG_FILE
-ENV LOG_FILE ${LOG_FILE}
-
-# Set this to change the default cached state directory
-ARG ZEBRA_CACHED_STATE_DIR
-ENV ZEBRA_CACHED_STATE_DIR ${ZEBRA_CACHED_STATE_DIR:-/var/cache/zebrad-cache}
-
-# Zebra automatically detects if it is attached to a terminal, and uses colored output.
-# Set this to 'true' to force using color even if the output is not a terminal.
-# Set this to 'false' to disable using color even if the output is a terminal.
-ARG LOG_COLOR
-ENV LOG_COLOR ${LOG_COLOR}
-
 # Expose configured ports
-
 EXPOSE 8233 18233 $RPC_PORT
 
-# Config location
-
-# Use a configurable dir and file for the zebrad configuration file
-ARG ZEBRA_CONF_DIR
-ENV ZEBRA_CONF_DIR ${ZEBRA_CONF_DIR:-/etc/zebra}
-
-ARG ZEBRA_CONF_FILE
-ENV ZEBRA_CONF_FILE ${ZEBRA_CONF_FILE:-zebrad.toml}
-
-ARG ZEBRA_CONF_PATH
-ENV ZEBRA_CONF_PATH ${ZEBRA_CONF_PATH:-$ZEBRA_CONF_DIR/$ZEBRA_CONF_FILE}
-
-# Other settings
-
-ARG SHORT_SHA
-ENV SHORT_SHA ${SHORT_SHA}
-
-# Set this to send sentry reports when Zebra crashes
-ARG SENTRY_DSN
-ENV SENTRY_DSN ${SENTRY_DSN}
```

Comment:
These variables control logging and panic reports:
- SHORT_SHA tells us the git commit in the logs and the sentry reports
- SENTRY_DSN sends crash reports to sentry, which sends a notification to Slack

The deployed Google Cloud instances should log both values when they start up. If that's not working, we'll see messages like "sentry is disabled", and log lines without any git commit hashes in them.

Reply:
Confirmed with deployed instance: https://console.cloud.google.com/compute/instancesDetail/zones/us-east1-c/instances/zebrad-fix-entrypoint-ports-ad5a3a4-mainnet?authuser=1&project=zfnd-dev-zebra